src/engraving/rendering/dev/pagelayout.cpp

```diff
@@ -433,11 +433,7 @@ void PageLayout::collectPage(LayoutContext& ctx)
     }
 
     for (const System* system : page->systems()) {
-        Fraction systemStartTick = system->measures().front()->tick();
-        Fraction systemEndTick = system->measures().back()->endTick();
-        if (systemEndTick > ctx.state().startTick() && systemStartTick <= ctx.state().endTick()) {
-            SystemLayout::centerElementsBetweenStaves(system);
-        }
+        SystemLayout::centerElementsBetweenStaves(system);
```

Comment:
Have you done any performance tests? Will this change have any effect on layout time?

Reply:
Layout time of the full score is totally unaffected by this (because it already does every system of every page, so this doesn't change anything). The change is only during editing operations on a page, and is absolutely minimal.